scripts/compileMessageBundles.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * The message bundles compiler script.
+ *
+ * This script compiles a mega message bundle which contains
+ * all messages for all available locales.
+ * Copyright 2018 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion");
+var gpii = fluid.registerNamespace("gpii");
+
+var fs = require("fs");
+var path = require("path");
+var shell = require("shelljs");
+
+require("./shared/messageBundlesCompiler.js");
+
+/**
+ * This function generates the messages bundle and writes into a given file.
+ * @param {String[]} messageDirs - An array of the directories from which messages
+ * files are to be retrieved.
+ * @param {String}  resultFilePath - The file where the bundles are going to be written.
+ */
+gpii.app.compileMessageBundles = function (messageDirs, resultFilePath) {
+    require("gpii-windows");
```

Comment:
We should also use `require("gpii-windows/index.js")` as `require("gpis-windows")` also runs gpii (see `gpii-windows/gpii.js`)
I just spent some time struggling with this...

Reply:
Thanks for your testing @danailbd. I've just added that change.